Kitodo/src/test/java/org/kitodo/production/services/data/ImportServiceTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+
+package org.kitodo.production.services.data;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.kitodo.production.dto.ProcessDTO;
+import org.kitodo.production.dto.ProjectDTO;
+
+public class ImportServiceTest {
```

Comment:
Can this new test class written with Junit5 usage instead of Junit4?

Reply:
Done.